examples/CalcQt/CMakeLists.txt

```diff
@@ -15,7 +15,9 @@ if(TARGET Qt::Core AND TARGET Qt::Gui AND TARGET Qt::Widgets AND TARGET Qt::Test
     add_executable(calcqt src/CalcQt.cpp)
     target_link_libraries(calcqt PRIVATE libcalcqt Qt::Widgets)
 
-    if(Qt::Test)
+    if(TARGET Qt::Test)
+        message (WARNING "QtTestCalculatorQtSteps is leaking (see issue #212)")
```

Comment:
Don't think this warning belongs to the build file. At this point we also don't know if the memory leak is in `QtTestCalculatorQtSteps` or in QtTest support.

Reply:
Warning removed.